dev/tools/controllerbuilder/pkg/llm/gemini.go

```diff
@@ -67,7 +71,7 @@ func (c *GeminiClient) StartChat(systemPrompt string) Chat {
 	model.SetTemperature(1)
 	model.SetTopK(40)
 	model.SetTopP(0.95)
-	model.SetMaxOutputTokens(8192)
+	model.SetMaxOutputTokens(65536)
```

Comment:
Is this the max for 2.0 or 2.5? I thought we previously used the max (but I don't remember if that's 1.5 or 2.0)